_includes/header.html

```diff
@@ -23,13 +23,11 @@
           <div class="grid-col-auto">
             {% image_with_class "./node_modules/@uswds/uswds/dist/img/us_flag_small.png" "usa-banner__header-flag" "U.S. flag" %}
           </div>
-          <div class="grid-col-fill tablet:grid-col-auto">
+          <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
```

Comment:
Because 'An official' sounds like 'Unofficial'. See 3rd bullet point in PR description.

Reply:
Actually, re-read my own link and this change is pertinent to the IDENTIFIER (footer), and the revision looks like this:
`<span aria-hidden="true">An </span>official website of the <a href="https://www.cisa.gov" class="usa-link">Cybersecurity and Infrastructure Security Agency</a>`. Having said that, [the official BANNER component](https://designsystem.digital.gov/components/banner/) markup matches my revision. I'm either missing something or USWDS has a bug in their banner component.

Since this matches the official code, I suggest we test the banner on our own and open up an issue with USWDS if we find that it's a bug.

Reply:
^ @SamiyahKey @abroddrick @lizpearl

Reply:
Tested with voiceover, no issues. USWDS has the hidden content linked to the 'Here's how you know' button.